server/controllers/entities/lib/resolver/create_entity_from_seed.js

```diff
@@ -46,6 +47,11 @@ const createEdition = async (edition, works, userId, batchId) => {
   // garantee that an edition shall not have label
   edition.labels = {}
 
+  if (enrich === true) {
+    const { url: imageHash } = await dataseed.getImageByIsbn(isbn)
+    if (imageHash) claims['invp:P2'] = [ imageHash ]
```

Comment:
this override any seed images provided, what about the case where images are provided in the seed, but not all of them, so we would like a soft update (only create if `invp:P2` doesnt exists) ?

Reply:
yes. but if an image was provided, it would probably rather be as a URL than an image hash, so we also need to add a step to allow conversion

Reply:
done in 396c917d and bbeb3235